### ISSUE TITLE
Doesn't work well with project/window changes

### ISSUE THREAD

**Issue by devurandom** [clojure-lsp/clojure-lsp-intellij#37]
I noticed that one Clojure window / project still shows that it is connecting to Clojure LSP, while another one is still "analyzing external classpath". I only see a single Clojure LSP process:
```
❯ ps -l | where name =~ "clojure-lsp"
  #    pid     ppid                                        name                                       ...
  0    12181   10403   [REDACTED]/.local/share/JetBrains/IntelliJIdea2023.3/clojure-lsp/clojure-lsp
```

Closing the window where I saw "analyzing external classpath" seems to have the other window disconnected from Clojure LSP and I get:
```
❯ ps -l | where name =~ "clojure-lsp"
 empty list
```

**Comment by ericdallo** [comment]
The problem is that we use a global atom for the plugin which I thought would be initiated per project, but it seems intellij uses it globally across windows as well, so we need to split that by project

**Comment by ericdallo** [comment]
It should be fixed on > 2.3.0
